scss/_patterns_subnav.scss

```diff
@@ -132,43 +132,43 @@
   }
 
   %subnav-item--light-theme {
-    background-color: map-get($colors--light-theme, background);
+    background-color: $colors--light-theme--background;
 
     &,
     &:active,
     &:focus &:visited {
-      color: map-get($colors--light-theme, text-default);
+      color: $colors--light-theme--text-default;
     }
 
     &:hover {
-      color: map-get($colors--light-theme, text-hover);
+      color: $colors--light-theme--text-default;
```

Comment:
Should this one be `--hover`?

Reply:
```suggestion
      color: $colors--light-theme--hover;
```

Reply:
it was changed to the default because I did the hover with transparency